### ISSUE TITLE
[BUG] Can't make skill checks

### ISSUE THREAD

**Issue by GambetTV** [sdenec/tidy5e-sheet#763]
**Module Version:** v0.8.41
**Foundry Version:** v10.291
**DnD5e Version:** v2.2.1

Probably just because it hasn't been updated for the latest DnD5e version, but you can't roll Skill Checks in the character sheets anymore. And I get the following console error when I try:

```
base-sheet.mjs:1355 Uncaught TypeError: undefined. Cannot read properties of null (reading 'dataset')
[Detected 1 package: system:dnd5e]
    at Tidy5eSheet._onRollSkillCheck (base-sheet.mjs:1355:60)
    at HTMLHeadingElement.dispatch (jquery.min.js:2:43064)
    at y.handle (jquery.min.js:2:41048)
```

**Comment by geoidesic** [comment]
Not fixed. I can confirm this is still happening even with version 0.8.41

**Comment by GambetTV** [comment]
He updated to 0.9.1. That update fixed it for me.

**Comment by geoidesic** [comment]
Fixed indeed in 0.9.1 🙏 tx
